.gitignore

```diff
@@ -1,12 +1,7 @@
-*.class
```

Comment:
No need to comment out specific file extensions. Everything in `target` folder is excluded from the source control.